example/road-traffic/road-traffic-demo/src/main/java/uk/gov/gchq/gaffer/traffic/Queries.java

```diff
@@ -61,7 +67,7 @@ private void run() throws OperationException, IOException {
         final Graph graph = createGraph(user);
 
         // Get the schema
-        System.out.println(graph.getSchema().toString());
+        LOGGER.info(graph.getSchema().toString());
```

Comment:
Will this be visible when run or does the class need the level set to INFO?

Reply:
As it will run in a `main` function

Reply:
The logger will exist (it is static) and so it should work. If there's no logger config and it defaults to not showing info logging that could be a problem. Can revert to how it was and add a suppression if you prefer. I think did that for another class which had `println` but no logger.

Reply:
Can you test it still outputs these when run?

Reply:
I decided it was easier to revert this and add an exception. So long as it really is being used as a standalone class with the main function, using `println` is not a genuine problem.